plotly/tools.py

```diff
@@ -2871,3 +2934,188 @@ def get_candle_decrease(self):
         return (decrease_x, decrease_close, decrease_dif,
                 stick_decrease_y, stick_decrease_x)
 
+class _Dendrogram(FigureFactory):
+    
+    """
+    Refer to FigureFactory.create_dendrogram() for docstring.
+    """
+
+    def __init__(self, X, orientation='bottom', labels=None, colorscale=None, \
```

Comment:
:cow2: (style tip) the `\` for a line break while inside parentheses is redundant.